vocab/template.html

```diff
@@ -0,0 +1,226 @@
+<html lang="en">
+  <head>
+    <meta charset='utf-8'/>
+    <title></title>
+    <script class="remove" src="https://www.w3.org/Tools/respec/respec-w3c"></script>
+    <script>
+      function remove_status_remark() {
+          const sotd = document.getElementById("sotd");
+          const p = sotd.getElementsByTagName('p')[0];
+          sotd.removeChild(p);
+      }
+    // Note: The vocabulary URL must be adapted for the given environment!!!!
+      function massageSVGLinks(utils, content, url) {
+        const retval = content
+          .replace('<svg', '<svg aria-details="#vocabulary-diagram-alt" ')
+          .replace(/xlink:href/g, 'href')
+          .replace(/href="https:\/\/www.w3.org\/ns\/credentials\/status-list#/g, 'href="#');
+        return retval;
+      }
+
+    </script>
+    <script class="remove">
+      var respecConfig = {
+        specStatus:  "base",
+        shortName:   "ns/credentials/status-list",
+        thisVersion: "https://www.w3.org/ns/credentials/status-list/",
```

Comment:
Just flagging this, so that we should not forget: these entries may have to change if the decision is to choose a different namespace URL.

Reply:
Fixed in aa07c21ffebac1c6456a98c0e4352be392bacd4a.